llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -1760,14 +1759,14 @@ bool FilterChooser::filterProcessor(bool AllowMixed, bool Greedy) {
   bool AllUseless = true;
   unsigned BestScore = 0;
 
-  for (unsigned i = 0, e = Filters.size(); i != e; ++i) {
-    unsigned Usefulness = Filters[i].usefulness();
+  for (const auto &[I, TFilter] : enumerate(Filters)) {
```

Comment:
Maybe use "Idx" instead of "I".
What's the "T" in "TFilter" mean?

Reply:
Done. Also `TFilter` renamed to `Filter` (agreed, looks better than the previous which meant "temp" filter).